_posts/en/newsletters/2020-09-30-newsletter.md

```diff
@@ -63,26 +63,28 @@ popular Bitcoin infrastructure software.
   meaning that syncing the Bitcoin block chain can require many millions of these
   elliptic curve point multiplications. Any technique to make point
   multiplications more efficient therefore has the potential to significantly
-  speed up Bitcoin's initial sync.
+  speed up Bitcoin Core's initial sync.
 
-    In 2011, [Hal Finney described][finney endomorphism] a way to more
+    In a [2011 bitcointalk post][finney endomorphism], Hal Finney described a
+    method by Gallant, Lambert and Vanstone (GLV) to
     efficiently compute elliptic curve point multiplications using an
     [endomorphism][] on the curve (a mapping from the curve to itself which
-    preserves all relationships between points). By using this endomorphism,
-    the multiplication can effectively be broken into two parts, calculated separately,
+    preserves all relationships between points). By using this GLV endomorphism,
+    the multiplication can be broken into two parts, which are calculated
+    simultaneously
     and then recombined to arrive at the solution. Doing this can reduce the
```

Comment:
There is no recombination step. It's a single computation that computes a*P+b*Q simultaneously.